grid/social-media/followers/profile.supr

```diff
@@ -64,6 +90,29 @@ usecase GetFollowers safe {
     A human-readable explanation specific to this occurrence of the problem.
     """
     detail
+
+    """
+    Code
+    Error code.
+    """
+    code! ErrorCode!
```

Comment:
I have added machine processable error code. Code is marked as required and non-nullable.